packages/mdc-segmented-button/segment/_segment.scss

```diff
@@ -24,12 +24,26 @@
 
 @use '@material/feature-targeting';
 @use '@material/typography/mixins' as typography-mixins;
+@use "@material/touch-target/mixins" as touch-target-mixins;
```

Comment:
Importing from `_mixins.scss` files are now deprecated. Most of the MDC base components (typography, touch-target, etc) have core sass modules that you can use them directly.

e.g., `@material/typography/typography`

Reply:
A couple, like elevation and ripple, have `elevation/elevation` and `elevation/elevation-theme`. Is it better to use both or just use `elevation`?